src/main/java/seedu/task/Event.java

```diff
@@ -0,0 +1,32 @@
+package task;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Event extends Task {
+
+    private final Date duration;
+    private final SimpleDateFormat ft = new SimpleDateFormat("dd MMM yyy h.mma");
```

Comment:
Same as above. Maybe consider changing variable name to match constant naming convention